tests/reference_exports/light/animation_various_lights.escn

```diff
@@ -56,7 +56,7 @@ tracks/0/keys = {
 	"times":PoolRealArray(0.0, 0.833333, 1.25, 1.91667),
 	"transitions":PoolRealArray(1, 1, 1, 1),
 	"update":0,
-	"values":[true, false, true, true]
+	"values":[true, true, true, true]
```

Comment:
This is unrelated to my light changes, and I get this diff even on master. I'm guessing that either this reference scene wasn't updated in the past, or there's some difference with my blender version.

Reply:
I think it is caused by you moved 
```
        light_node['shadow_enabled'] = light.shadow_method != "NOSHADOW"
        light_node['light_negative'] = light.use_negative
```
to
```
        AttributeConvertInfo(
            'shadow_method', 'shadow_enabled', lambda x: x != "NOSHADOW"
        ),
        AttributeConvertInfo('use_negative', 'light_negative', lambda x: x),
```
The `AttributeConvertInfo` is also used for animation exporting, the lambda also convert animation value from blender to godot.   The  `shadow_enabled` and `light_negative` in animation can not be handled the same way as the node attribute, so they are previously not written as `AttributeConvertInfo`.  I think you'd better keep what it was to avoid breaking this example.